junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/EnumSource.java

```diff
@@ -56,7 +58,7 @@
 	 * @see #names
```

Comment:
Probably good to document here what the semantics of the `default` value are.

Reply:
Done in 47aa8af2cd282a53790de0bb05edd7efaec14f2c.